Core/HLE/sceKernelThread.cpp

```diff
@@ -1462,6 +1468,12 @@ void __KernelLoadContext(PSPThreadContext *ctx, bool vfpuEnabled) {
 		memcpy(currentMIPS->vfpuCtrl, ctx->vfpuCtrl, sizeof(ctx->vfpuCtrl));
 	}
 
+#ifdef _DEBUG
```

Comment:
This check is cheap enough that we might as well keep it in release too?

Reply:
Well, was just being paranoid, but sure I'll remove the check.

-[Unknown]